integration/spark2/src/test/scala/org/apache/spark/carbondata/TestStreamingTableOperation.scala

```diff
@@ -123,12 +123,9 @@ class TestStreamingTableOperation extends QueryTest with BeforeAndAfterAll {
 
     createTable(tableName = "agg_table", streaming = true, withBatchLoad = false)
 
-    var csvDataDir = new File("target/csvdatanew").getCanonicalPath
+    val csvDataDir = integrationPath + "target/csvdatanew"
```

Comment:
it should be `integrationPath + "/spark2/target/csvdatanew`

Reply:
fixed